src/test/java/frc/pathrecogonizer/TestVisionFieldLayoutRecognizer.java

```diff
@@ -0,0 +1,31 @@
+package frc.pathrecogonizer;
+
+import org.junit.Test;
+import java.util.List;
+import java.util.ArrayList;
+import static org.junit.Assert.assertEquals;
+
+public class TestVisionFieldLayoutRecognizer {
```

Comment:
good! you need LOTS more tests though. i expect one for each range, plus some testing of happens when a value is not in any range, etc.